pkg/insights/resyncer.go

```diff
@@ -270,7 +270,7 @@ func (r *resyncer) Start(stop <-chan struct{}) error {
 
 					externalServices := &core_mesh.ExternalServiceResourceList{}
 					if err := r.rm.List(tenantCtx, externalServices, store.ListByMesh(event.mesh)); err != nil {
-						log.Error(err, "unable to get ExternalServices to recompute insights")
+						log.Error(err, "unable to get ExternalServices to recompute insights", "event", event)
```

Comment:
We don't need anymore, because we are adding it here https://github.com/kumahq/kuma/pull/7807/files#diff-977161303ba444a80ba80bd6a3446bb1657906e1efa0d54fdc9b36f714fb9cc6R264